config/settings.yml.example

```diff
@@ -56,3 +56,10 @@
 # Uncomment and modify if you want to change the log level
 # WARN, DEBUG, ERROR, FATAL, INFO, UNKNOWN
 #:log_level: ERROR
+
+# Log buffer size and extra buffer size (for errors). Defaults to 1500 messages in total,
+# which is about 250 kB request.
+#:log_buffer: 1000
+#:log_buffer_errors: 500
+# Log buffer threshold (debug messages are ignored by default)
+#:log_buffer_level: INFO
```

Comment:
@lzap: I still do not understand the need for specifying log level in both the buffer and the logger. This could lead to a situation when 'debug' was specified for the logger, 'info' for the buffer, resulting in no debug-level information making it to the logger.